src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -204,25 +204,24 @@ public void visitToken(DetailAST ast) {
             && ast.findFirstToken(TokenTypes.LITERAL_IF) != null) {
             isElseIf = true;
         }
-        final boolean isDefaultInAnnotation = isDefaultInAnnotation(ast);
+        final boolean isInAnnotationField = isInAnnotationField(ast);
         final boolean skipStatement = isSkipStatement(ast);
         final boolean skipEmptyLoopBody = allowEmptyLoopBody && isEmptyLoopBody(ast);
 
-        if (slistAST == null && !isElseIf && !isDefaultInAnnotation
+        if (slistAST == null && !isElseIf && !isInAnnotationField
                 && !skipStatement && !skipEmptyLoopBody) {
             log(ast.getLineNo(), MSG_KEY_NEED_BRACES, ast.getText());
         }
     }
 
     /**
-     * Checks if ast is the default token of an annotation field.
+     * Checks if ast is in an annotation field.
      * @param ast ast to test.
-     * @return true if current ast is default and it is part of annotation.
+     * @return true if current ast is part of an annotation field.
      */
-    private static boolean isDefaultInAnnotation(DetailAST ast) {
+    private static boolean isInAnnotationField(DetailAST ast) {
```

Comment:
There are no annotation fields which validate a brace being needed, so checking for `default` isn't needed and was removed thereby causing a bunch of code to be changed.